From 450ee52bc49d934ee1086a27fca3b2d6d593a731 Mon Sep 17 00:00:00 2001
From: Francisco Boni <boboniboni@gmail.com>
Date: Sun, 17 Jan 2021 05:07:21 -0300
Subject: [PATCH 1/2] Fix testing on mock

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 60477ba..8ffbd73 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -173,7 +173,7 @@ find_package(PythonInterp)
 if(PYTHONINTERP_FOUND)
   enable_testing()
   add_test(NAME iwyu_tests
-    COMMAND ${PYTHON_EXECUTABLE} run_iwyu_tests.py -- $<TARGET_FILE:include-what-you-use>
+    COMMAND ${PYTHON_EXECUTABLE} run_iwyu_tests.py -- /usr/bin/include-what-you-use
     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
   add_test(NAME fix_includes_test
     COMMAND ${PYTHON_EXECUTABLE} fix_includes_test.py
-- 
2.28.0

